src/components/FixedGrid/Grid.js

```diff
@@ -24,8 +28,121 @@ function Grid(props) {
 
   const gridClassName = classNames(styles.grid, className);
 
+  const handleOnKeyUp = event => {
+    const keycode = event.code;
+    const { columns, rows } = other;
+    if (event.repeat) return;
+
+    const compareKeys = code => {
+      const right = code === 'ArrowRight';
+      const left = code === 'ArrowLeft';
+      const up = code === 'ArrowUp';
+      const down = code === 'ArrowDown';
+
+      if (!(right || left || up || down)) return true;
+      event.preventDefault();
+
```

Comment:
If you move the conditional `if (event.repeat) return;`  used on line 34, dont need to add the listener for prevent scroll on line 136

Reply:
Appear a good change to only prevent event repeat for arrow keys. 
the only against with remove the event listener is when the user clicks on the navbar. By pressing arrow keys scrolling occurs. 
If we want to save this behavior, we should prevent set focus on the first tile when the focus is on the Board body. 
if not we should preserve the event listener